### ISSUE TITLE
Storyshots setup

### ISSUE THREAD

**Issue by dankellett** [nuxt-modules/storybook#210]
I've been attempting to setup Storyshots with this module and I'm running into an issue that I'm currently blocked on and would love some help getting around. I've created a demo repo that reproduces the issue.

You can find the storyshot test in the `\test` folder and can run the test using `npm test` after restoring packages.

https://github.com/dankellett/nuxt-storyshots

The error is generated when the resolver hits the following lines in `.nuxt-storybook/index.js`
```
Cannot find module 'nuxt_plugin_plugin_3fb405da' from '.nuxt-storybook/index.js'

13 | /* Plugins */
14 | 
15 | import nuxt_plugin_plugin_3fb405da from 'nuxt_plugin_plugin_3fb405da' // Source: .\\components\\plugin.js (mode: 'all')
   | ^
16 | import nuxt_plugin_plugin_533a0fc1 from 'nuxt_plugin_plugin_533a0fc1' // Source: .\\vuetify\\plugin.js (mode: 'all')
```

**Comment by blowsie** [comment]
> Does that mean, that there is no support for storyshots at all? I'm currently also running into that issue and would love to use this project

For what its worth, Storybook 8 is dropping support for storyshots.
https://storybook.js.org/blog/storybook-8-beta/

**Comment by tobiasdiez** [comment]
v4 of this module is no longer actively supported. Please try the newest version and open an new issue if the problem persists. Thank you for your understanding.
